src/message-stream.ts

```diff
@@ -181,29 +140,25 @@ export class MessageStream extends PassThrough {
    * @param {error?} err An error to emit, if any.
    * @private
    */
-  destroy(err?: Error): void {
+  _destroy(err: null | Error, callback: (err?: null | Error) => void) {
```

Comment:
Just to double check - is this a method that leaks externally?  Is it possible this is a breaking change?

Reply:
Nope, this is only used internally. I can open a new PR for this portion, but this was essentially a destroy shim for v6.